### ISSUE TITLE
[BITV] 9.1.4.1/9.1 - Links in the documentation are differentiated from the surrounding text by both color and a symbol. However, the contrast ratio between the link color (blue) and the surrounding text color is below 3:1 and the symbol does not meet the contrast requirements for informative graphic elements. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/documentation#9673]
- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-45ae33ca9f-v9-n1

</details>
